### PR TITLE
styled the property upload page for a user. still need to add amenities

### PR DESCRIPTION
**Pull request by adamhundley**
I styled the property listings page and made it more user-friendly.
still need to add actual amenities for a user to select
also added a possible logo

### CODE REVIEW

File: app/controllers/user/properties_controller.rb [hunk 1]
```diff
@@ -4,6 +4,7 @@ def new
   end
 
   def create
+    require "pry"; binding.pry
```
Comment (Salvi6God) [inline app/controllers/user/properties_controller.rb]: @adamhundley you left in a pry here dude.

**Comment by Salvi6God** [comment]
@adamhundley everything looks good just get that pry out and I'll merge to master :+1:

**Comment by adamhundley** [comment]
nice catch dude. i removed it and rebased

**Comment by Salvi6God** [comment]
Awesome! I'll go ahead and merge now.

**Pull request merged**